doc/usermanual/slideshow/slideshow.xml

```diff
@@ -82,7 +82,7 @@
             </row>
             <row>
               <entry>
-                <emphasis>space</emphasis>
+                <emphasis>Space bar</emphasis>
```

Comment:
Same as above.

Reply:
The English Wikipedia distinguishes between the space key (spacebar) and the space character

https://en.wikipedia.org/wiki/Space_(punctuation)
https://en.wikipedia.org/wiki/Space_bar

Reply:
It also has https://en.wikipedia.org/wiki/Alt_key and https://en.wikipedia.org/wiki/Control_key.

Reply:
Maybe I do not have a clue today.
I think we're talking about different things. In the manual, we should replace space with spacebar, because we are talking about the key and not the character. According to wikipedia, space is the character that is written when the spacebar is pressed. 
The keys Ctrl, Alt and Shift are function keys that do not write any characters but only make a function e.g. to write an character as capital.

Long story short, in my opinion it is right when we write:

    It is turned on/off with key accelerator Ctrl + F

or

    Mouse scrolling while holding the control key pressed gives an extended zoom range between 2:1 and 1:10.

While writing this comment I noticed that sentences like these should be adapted (I can still adapt this):

    Fully zoomed-in image view while holding down the Ctrl + Z key with indication of the sharp areas in focus.

to

    Fully zoomed-in image view while holding down the key sequence Ctrl + Z with indication of the sharp areas in focus.

Reply:
My point was, that we should just use "space" when showing shortcuts.

That other sentence could also be written as "Fully zoomed-in image view while holding down Ctrl + Z with indication of the sharp areas in focus.". I am not a native speaker though.